main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func hello(w http.ResponseWriter, req *http.Request) {
+	fmt.Fprintf(w, "hello\n")
```

Comment:
Reply just with time.Now().String()

Reply:
Changed fmt.Fprintf to work with a channel